### PR TITLE
Created main page for the user interface

### PR DESCRIPTION
**Pull request by antoninguerre**

### CODE REVIEW

**Comment by antoninguerre** [comment]
Closes #11

File: PostalRateCalculator-Backend/src/test/java/PostalRateCalculator/PostalRateCalculatorApplicationTest.java [hunk 1]
```diff
@@ -7,6 +7,6 @@
 public class PostalRateCalculatorApplicationTest {
 
     @Test
-    public void contextLoads() {}
+    private void contextLoads() {}
```
Comment (JulietteDebray) [inline PostalRateCalculator-Backend/src/test/java/PostalRateCalculator/PostalRateCalculatorApplicationTest.java:10]: why

File: PostalRateCalculator-Backend/src/test/java/PostalRateCalculator/model/EnvelopeTest.java [hunk 2]
```diff
@@ -24,7 +24,7 @@ public void clearDatabase() {
     }
 
     @Test
-    public void testEnvelope() {
+    private void testEnvelope() {
```
Comment (JulietteDebray) [inline PostalRateCalculator-Backend/src/test/java/PostalRateCalculator/model/EnvelopeTest.java:27]: also why

File: PostalRateCalculator-Backend/src/test/java/PostalRateCalculator/PostalRateCalculatorApplicationTest.java (hunk 1 above)
Comment (antoninguerre) [reply PostalRateCalculator-Backend/src/test/java/PostalRateCalculator/PostalRateCalculatorApplicationTest.java:10]: comme ca it doesnt run en meme tps que les autres . That way we have the same number of tests as documented entries

File: PostalRateCalculator-Backend/src/test/java/PostalRateCalculator/model/EnvelopeTest.java (hunk 2 above)
Comment (antoninguerre) [reply PostalRateCalculator-Backend/src/test/java/PostalRateCalculator/model/EnvelopeTest.java:27]: same reason

Comment (antoninguerre) [reply PostalRateCalculator-Backend/src/test/java/PostalRateCalculator/model/EnvelopeTest.java:27]: on les mettra tous public a la fin

File: PostalRateCalculator-Frontend/src/components/Test.vue [hunk 3]
```diff
@@ -0,0 +1,73 @@
+<template>
```
Comment (JulietteDebray) [inline PostalRateCalculator-Frontend/src/components/Test.vue:1]: what is Test.vue for?

**Review by JulietteDebray — COMMENTED** [review]
all good, just not sure what test.vue is for

File: PostalRateCalculator-Frontend/src/components/Test.vue (hunk 3 above)
Comment (antoninguerre) [reply PostalRateCalculator-Frontend/src/components/Test.vue:1]: that wa me testing different layouts. I can't safely delete it from my branch i have to wait till it's on master

**Pull request merged**